tests/tracing/instrumentation/test_action_executor.py

```diff
@@ -0,0 +1,58 @@
+from typing import Any, Dict, Sequence, Text, Optional
+
+import pytest
+from opentelemetry.sdk.trace import ReadableSpan, TracerProvider
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
+
+from rasa_sdk.tracing.instrumentation import instrumentation
+from tests.tracing.instrumentation.conftest import MockActionExecutor
+from rasa_sdk.types import ActionCall
+from rasa_sdk import Tracker
+
+
+@pytest.mark.parametrize(
+    "action_name, expected",
+    [
+        ("check_balance", {"action_name": "check_balance", "sender_id": "test"}),
+        (None, {"sender_id": "test"}),
```

Comment:
This should be a None converted to string, otherwise opentelemetry will issue the type warnings.

Reply:
This is the only the value of the actions, not the trace tag. We're not extracting the attribute if it's not available. That's why the action_name key is not in the dict.

Reply:
Got it! All clear now.